code/modules/games/tarot.dm

```diff
@@ -0,0 +1,43 @@
+/* this is a playing card deck based off of the Rider-Waite Tarot Deck.
+*/
+
+/obj/item/weapon/deck/tarot
+	name = "deck of tarot cards"
+	desc = "For all your occult needs!"
+	icon_state = "deck_tarot"
+
+/obj/item/weapon/deck/tarot/New()
+	..()
+
+	var/datum/playingcard/P
+	for(var/name in list("Fool","Magician","High Priestess","Empress","Emperor","Hierophant","Lovers","Chariot","Strength","Hermit","Wheel of Fortune","Justice","Hanged Man","Death","Temperance","Devil","Tower","Star","Moon","Sun","Judgement","World"))
+		P = new()
+		P.name = "[name]"
+		P.card_icon = "tarot_major"
+		P.back_icon = "card_back_tarot"
+		cards += P
+	for(var/suit in list("wands","pentacles","cups","swords"))
+		for(var/number in list("ace","two","three","four","five","six","seven","eight","nine","ten","page","knight","queen","king"))
+			P = new()
+			P.name = "[number] of [suit]"
+			P.card_icon = "tarot_[suit]"
+			P.back_icon = "card_back_tarot"
+			cards += P
+
+/obj/item/weapon/deck/tarot/shuffle()
```

Comment:
Can't this just use the existing shuffle proc for decks instead of having its own one?